manifests/m/Microsoft/store/12105.1001.23.0/Microsoft.store.installer.yaml

```
@@ -0,0 +1,36 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.store
+PackageVersion: 12105.1001.23.0
+Installers:
+- Platform:
+  - Windows.Universal
+  MinimumOSVersion: 10.0.16299.0
+  Architecture: arm
+  InstallerType: msix
+  InstallerUrl: http://tlu.dl.delivery.mp.microsoft.com/filestreamingservice/files/a3a0623b-5e55-4a17-9de6-e4eeb1de576f?P1=1627085385&P2=404&P3=2&P4=E6q7vN6vOgoX72kHnXrokJwwqJrT7q9oa4BfexRt%2fB51HlM4kCF68WSCn0HOXm%2bxArjhMIGqY8iBsRfa3HD3ew%3d%3d
+  InstallerSha256: A9B0BC538A529720029C7FE8DEC9FA8B74022E7940B7EA3D516E28D2F4B5714F
+  SignatureSha256: E10E677C62CF01EB490080F46FDB64508316911E2D9F82A76F598066EAAA6DBB
+  PackageFamilyName: Microsoft.WindowsStore_8wekyb3d8bbwe
+- Platform:
+  - Windows.Universal
+  MinimumOSVersion: 10.0.16299.0
+  Architecture: x64
+  InstallerType: msix
+  InstallerUrl: http://tlu.dl.delivery.mp.microsoft.com/filestreamingservice/files/a3a0623b-5e55-4a17-9de6-e4eeb1de576f?P1=1627085385&P2=404&P3=2&P4=E6q7vN6vOgoX72kHnXrokJwwqJrT7q9oa4BfexRt%2fB51HlM4kCF68WSCn0HOXm%2bxArjhMIGqY8iBsRfa3HD3ew%3d%3d

```

```suggestion
  InstallerUrl: http://tlu.dl.delivery.mp.microsoft.com/filestreamingservice/files/a3a0623b-5e55-4a17-9de6-e4eeb1de576f?P1=1627085385&P2=404&P3=2&P4=E6q7vN6vOgoX72kHnXrokJwwqJrT7q9oa4BfexRt%2fB51HlM4kCF68WSCn0HOXm%2bxArjhMIGqY8iBsRfa3HD3ew%3d%3d
```
These are temporary URLs and will expire.